supervisor_docker.conf

```diff
@@ -1,5 +1,20 @@
+[unix_http_server]
+file=/tmp/supervisor.sock
+chmod=0700
+
+[supervisorctl]
+serverurl=unix:///tmp/supervisor.sock
+
+[rpcinterface:supervisor]
+supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
+
```

Comment:
I added this so that i can see what process supervisor running. Open for changing it if it is redundant

`supervisorctl -c /code/supervisor_docker.conf status`